frontend/src/components/AppToolBar/index.tsx

```diff
@@ -0,0 +1,75 @@
+import play from '../../img/play.svg';
+import useSWR from 'swr';
+import { fetchCustomData } from './api';
+import React, { useState } from 'react';
+import { Form, FormControl, ListGroup } from 'react-bootstrap';
+
+export const AppToolBar: React.FC = () => {
+    const { data, error } = useSWR('/api/stands/', fetchCustomData);
+    const [selectedItem, setSelectedItem] = useState('');
+    const [showList, setShowList] = useState(true);
+    const [hoveredItem, setHoveredItem] = useState(null);
+    if (!data) {
+        return (
+            <div className="alert alert-primary" role="alert">
+                Data is loading
+            </div>
+        );
+    }
+    if (error) {
+        return (
+            <div className="alert alert-danger" role="alert">
+                Cant load data {error.message}
+            </div>
+        );
+    }
+    const handleHover = (item) => {
+        setHoveredItem(item);
+    };
+    const handleSearch = (e) => {
+        setSelectedItem(e.target.value);
+        setShowList(true);
+    };
+    const filteredData = data.stands
+        .filter((item) => item['name'].toLowerCase().includes(selectedItem.toLowerCase()))
```

Comment:
Не критично, но точечная нотация будет нагляднее.
```suggestion
        .filter((item) => item.name.toLowerCase().includes(selectedItem.toLowerCase()))
```